package.json

```diff
@@ -144,19 +146,12 @@
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
-    },
-    "commitTypeMap": {
-      "docs": null,
-      "fix": "patch",
-      "perf": "patch",
-      "feat": "minor",
-      "refactor": null,
-      "release": "minor",
-      "ci": null,
-      "tests": null
     }
   },
   "release": {
-    "analyzeCommits": "@bubltechnology/customizable-commit-analyzer"
+    "verifyConditions": [
```

Comment:
need to override this to exclude `@semantic-release/condition-travis` - we use build stages to ensure the release only happens on the right branch, after after all tests passed etc.